### ISSUE TITLE
Errror: Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1

### ISSUE THREAD

**Issue by dcharrance** [HGLab/VIC#6]
Command: java -jar VIC-1.0-jar-with-dependencies.jar -annotate_variation /home/dcharrance/annovar/annotate_variation.pl -b hg19 -convert2annovar /home/dcha
rrance/annovar/convert2annovar.pl -table_annovar /home/dcharrance/annovar/table_annovar.pl -d /home/dcharrance/annovar/humandb -db /home/dcharrance/vicdb -i VCFprova.vcf -i
nput_type vcf -o prova.vic

Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
at vic.VIC.Read_dataset(VIC.java:461)
at vic.VIC.main(VIC.java:350)

Can you help me?
Thanks

Debora

**Comment by maxh190** [comment]
Hi Debora,

I have never seen that exception before.  Can you send me a small dataset
(including 3 or 4 variants) of your data (or revised data with the same
format) so that I can debug the issue?


Best,

Max

**Comment by dcharrance** [comment]
Hi Max,
Thanks for your answer!
Attached to this mail I send you my VCF file.

Best regards
Debora

**Comment by maxh190** [comment]
Did you attach your VCF file?  I didn't see it.

**Comment by dcharrance** [comment]
Yes, I did. I attached the file also in this email. Can you see it?

**Comment by dcharrance** [comment]
[VCF prova.txt](https://github.com/HGLab/VIC/files/5850200/VCF.prova.txt)
My VCF

**Comment by maxh190** [comment]
Gmail might block users to send a .vcf file.  Finally, I got your text vcf
file and will test it on the weekend.

**Comment by dcharrance** [comment]
Hi Max,

Perfect! Thanks to the availability. 
We can update on Monday.
Have a nice weekend

Debora
